src/ripple/app/main/Application.cpp

```diff
@@ -1344,26 +1344,34 @@ ApplicationImp::setup()
 
     // Configure the amendments the server supports
     {
-        auto const& sa = detail::supportedAmendments();
-        std::vector<std::string> saHashes;
-        saHashes.reserve(sa.size());
-        for (auto const& name : sa)
-        {
-            auto const f = getRegisteredFeature(name);
-            BOOST_ASSERT(f);
-            if (f)
-                saHashes.push_back(to_string(*f) + " " + name);
-        }
-        Section supportedAmendments("Supported Amendments");
-        supportedAmendments.append(saHashes);
+        auto buildAmendmentList =
+            [](Section section, std::vector<std::string> const& amendments) {
+                std::vector<std::string> hashes;
+                hashes.reserve(amendments.size());
+                for (auto const& name : amendments)
+                {
+                    auto const f = getRegisteredFeature(name);
+                    // BOOST_ASSERT(f);
```

Comment:
Why is this commented?

Reply:
IIRC, it's because I was hitting it during some testing with bogus feature names. Looks like, I forgot to uncomment it. Fixed.

Reply:
I don't see a fix pushed to github yet